standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -1603,21 +1645,24 @@ public void commitTxn(CommitTxnRequest rqst) throws NoSuchTxnException, TxnAbort
         }
 
         if (!isHiveReplTxn) {
-          createCommitNotificationEvent(dbConn, txnid , txnType);
+          createCommitNotificationEvent(jdbcTemplate, txnid , txnType);
```

Comment:
sh9uld it be dataSourceWrapper.getConnection()?

Reply:
done